react/components/SearchQuery.js

```diff
@@ -188,14 +188,22 @@ const useQueries = (
     },
   })
 
-  const { data: { sponsoredProducts } = [] } = useQuery(
+  const { data: { sponsoredProducts } = [], loading: sponsoredProductLoading } = useQuery(
```

Comment:
By the way, in order to get the error state as I commented below, you can do:

```suggestion
  const { data: { sponsoredProducts } = [], loading: sponsoredProductsLoading, error: sponsoredProductsError } = useQuery(
```

After that, the error will be available in the `sponsoredProductsError` variable.